jenkinsfile-runner-tests/tests.sh

```diff
@@ -12,11 +16,69 @@ oneTimeSetUp() {
 
   echo "Moving CLI jar file to $current_directory"
   find $test_framework_directory/source/custom-war-packager-cli/target -name '*-with-dependencies.jar' -exec mv {} $current_directory/cwp.jar ';'
+  cwp_jar="$current_directory/cwp.jar"
+}
+
+#
+# Use a JCasC configuration to build the JFR image, then get the system message that was set in
+# casc.yml to verify that it was applied.
+#
+# TODO for this and other tests we are copying casc.yml to the current directory so that it is
```

Comment:
Can't you use a relative path? I mean: something similar to https://github.com/jenkinsci/jenkinsfile-runner/blob/0a67d41049c882f5dd834b736210d943d5412cbb/tests/test_resources/cwp-produced-images/config/packager-config.yml#L15